File: Cenfomon/Cenfomon_COI/bl/entities/Answer.ts

export class Answer {
 // Atributos
  private _id: number;
  //private _options: Map <string, Dialog>;
//constructor 

  constructor(id: number) {
    this._id = id;
    //this._options = options;
  }

//get & set


  getid(): number {
    return this._id;
  }

  setid(value: number): void {
    this._id = value;
  }

//  get options(): Map<string, Dialog> {
//    return this._options;
//  }

//  set options(value: Map<string, Dialog>) {
//    this._options = value;
//  }
}

File: Cenfomon/Capa_Logica/src/main/java/com/cenfomon/Cenfomon_Capa_Logica/bl/entities/Answer.java
package com.cenfomon.Cenfomon_Capa_Logica.bl.entities;

import java.awt.*;
import java.util.Map;

public class Answer {
    /**
     * Atributos
     */
    int id;
    Map<String, Dialog> options;
    /**
     * get and set
     */
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Map<String, Dialog> getOptions() {
        return options;
    }

    public void setOptions(Map<String, Dialog> options) {
        this.options = options;
    }
}
